hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/RecordStore.java

```diff
@@ -292,6 +294,8 @@ boolean merge(MapMergeTypes<Object, Object> mergingEntry,
 
     void forEach(BiConsumer<Data, Record> consumer, boolean backup, boolean includeExpiredRecords);
 
+    Iterator<Map.Entry<Data, Record>> iterator();
```

Comment:
Looks like the same can be achieved with `RecordStore.getStorage().mutationTolerantIterator()`, it's also clearly indicates what kind of iterator we are getting.

Reply:
Fixed.